opensearch-observability/src/main/kotlin/org/opensearch/observability/action/ObservabilityActions.kt

```diff
@@ -30,6 +31,12 @@ internal object ObservabilityActions {
     fun create(request: CreateObservabilityObjectRequest, user: User?): CreateObservabilityObjectResponse {
         log.info("$LOG_PREFIX:ObservabilityObject-create")
         UserAccessManager.validateUser(user)
+        if (!UserAccessManager.doesUserHasAccessToCreate(user)) {
+            throw OpenSearchStatusException(
+                "Permission denied for ObservabilityObject creation",
```

Comment:
I noticed in the exceptions below you are leaving it really generic i.e. "Permission denied for ObservailibtyObject ${id}", should this be the same?